### ISSUE TITLE
A precompiled iOS application file

### ISSUE THREAD

**Issue by mr-cringe61** [liftoff-app/liftoff#85]
I would love to see a precompiled .ipa for iOS devices. There are already precompiled files for Android, Linux and Windows and it would be quite useful. Not much to say otherwise. Looks great. ;)

**Comment by zachatrocity** [comment]
Yes! This is in progress. I think once you have that .ipa you could use something like AltStore to install right?

**Comment by mr-cringe61** [comment]
Exactly! Although I don’t know the minimum required version yet (didn’t find it in the Info.plist), I thought I can install it with TrollStore (permasigned App Installer for iOS 15.0-15.4.1 that works without a Jailbreak) on my device. For other people it would probably be AltStore, Sideloadly or Testflight (if you want to submit your app there). And well, the lower the minimum supported version the better.

**Comment by zachatrocity** [comment]
It's already on test flight. Reach out to @mykdavies for details. 

TestFlight links in here:

https://lemmy.world/post/527432
